src/Features/LanguageServer/Protocol/DefaultCapabilitiesProvider.cs

```diff
@@ -72,26 +72,26 @@ public ServerCapabilities GetCapabilities(ClientCapabilities clientCapabilities)
 
             capabilities.HoverProvider = true;
 
+            capabilities.SemanticTokensOptions = new SemanticTokensOptions
```

Comment:
FYI @nohwnd - when this PR merges and you update roslyn packages, the custom handling on line 161 in JsonRpcServer can be removed